cve_bin_tool/cvedb.py

```diff
@@ -268,10 +269,7 @@ def get_cves(self, vendor, product, version):
         query = """SELECT CVE_number FROM cve_range WHERE
         vendor=? AND product=? AND version=?"""
         cursor.execute(query, [vendor, product, version])
-        # FIXME: this seems inefficient
-        cve_list = []
-        for cve in cursor:
-            cve_list.append(cve[0])
+        cve_list = list(map(lambda x: x[0], cursor.fetchall()))
```

Comment:
Ah, there we go.  I'd have to check to see if it's really more efficient in practice at runtime, but it certainly *looks* nicer.